<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myWidget</class>
 <widget class="QMainWindow" name="myWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenFile">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePattern">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAlgorithm">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>bruteForce</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="tbLog">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>671</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetResults">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>671</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
